pages/management/applications.md

```diff
@@ -19,12 +19,12 @@ Here we have an application named "myFleet" and currently it only has one device
 
 When you create an application a special resin.io operating system is generated specifically for that application and its associated device type.
 
-When you install this image onto your device it will automatically appear in your application dashboard, no manual intervention is required. You can copy this one download to multiple SD cards and resin.io will associate all these devices with their own unique ID and fancy name.
+When you install this image onto your device it will automatically appear in your application dashboard, no manual intervention is required. You can install this one downloaded image file to multiple SD cards and resin.io will associate all these devices with their own unique ID and fancy name.
 
 
 ### Deploying Your Code to an Application
 
-The key thing to know about any application from your perspective as a developer is it's git endpoint - this is visible in the applications list on the [applications dashboard](http://dashboar.resin.io) and also in the top-right hand corner of each individual application dashboard.
+The key thing to know about any application from your perspective as a developer is it's git endpoint - this is visible in the applications list on the [applications dashboard](http://dashboard.resin.io) and also in the top-right hand corner of each individual application dashboard.
```

Comment:
nice typo catch! again to avoid this kind of errors you can use the Handlebars interpolation as mentioned in my other comment.